receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -112,15 +115,28 @@ const (
 
 // New creates a TraceReceiver that receives traffic as a collector with both Thrift and HTTP transports.
 func New(ctx context.Context, config *Configuration, nextConsumer consumer.TraceConsumer, logger *zap.Logger) (receiver.TraceReceiver, error) {
-	return &jReceiver{
+	jR := &jReceiver{
 		config:          config,
 		defaultAgentCtx: observability.ContextWithReceiverName(context.Background(), "jaeger-agent"),
 		nextConsumer:    nextConsumer,
 		tchanServer: &jTchannelReceiver{
 			nextConsumer: nextConsumer,
 		},
 		logger: logger,
-	}, nil
+	}
+
+	if config.RemoteSamplingEndpoint != "" {
+		// Create upstream grpc client
+		conn, err := grpc.Dial(config.RemoteSamplingEndpoint, grpc.WithInsecure())
```

Comment:
Is it safe to assume that it will always be insecure? It doesn't need to be addressed in this PR but may need an issue to not lose track of it.

Reply:
@joe-elliott has a good point. The creation of receiver is typically a phase that is distinct from starting. First all receivers are created and then they are started and start step is preceded by a log message. For uniformity and to ensure the actual functionality matches what is printed in the logs it is best to perform this operation in `StartTraceReception`. The config is available in `StartTraceReception` since it is a field of this struct, that should not be a problem.